katello/hooks/post/09-mongo_syspath.rb

```diff
@@ -0,0 +1,17 @@
+def install_syspaths
+  unless facts[:operatingsystemmajrelease] == '7'
+    `rpm -qa | grep rh-mongodb34-syspaths`
+    if $?.success?
+      logger.debug 'rh-mongodb34-syspaths already installed, skipping.'
+    else
+      logger.warn 'rh-mongodb34-syspaths not present, installing.'
```

Comment:
I don't see a reason to make this a warning. It's part of the normal operation so info should suffice.

```suggestion
      logger.info 'rh-mongodb34-syspaths not present, installing.'
```